frontend/src/utils/formValidation.js

```diff
@@ -31,3 +31,16 @@ export const loginValidationSchema = yup.object().shape({
     .required("Email is required"),
   password: yup.string().required("Password is required"),
 });
+
+/*export function newBidValidationSchema(minValue) {
+  return yup.object().shape({
+    newBid: yup
+      .number()
+      .min(minValue, `Your bid cannot be lower than ${minValue}`)
+      .required("New Bid is required"),
+  });
+}*/
+
+export const newBidValidationSchema = yup.object().shape({
+  amount: yup.number().required(),
```

Comment:
Think about the max value that should be allowed for number fields. We don't want to cause overflow.